docs/docs/installation/running-on-kubernetes.mdx

```diff
@@ -92,6 +92,34 @@ postgresql:
   postgresqlPassword: superset
 ```
 
+Make sure you are giving your own SECRET_KEY for the encryption instead of default SECRET_KEY.
```

Comment:
Consider replacing by:
"""
Make sure, you set a unique strong complex alphanumeric string for your `SECRET_KEY` and use a tool to help you generate a sufficiently random sequence
"""

Reply:
Done.